src/lib/userStorage/FeedStorage.js

```diff
@@ -758,37 +767,18 @@ export class FeedStorage {
    * @param {*} event
    */
   async getFromOutbox(event: FeedEvent) {
-    let senderPubkey = await this.userStorage.getUserProfilePublickey(get(event, 'data.receiptEvent.from'))
-    let recipientPubkey = this.gunuser.is.pub
-
-    if (!senderPubkey) {
-      log.warn('getFromOutbox sender not found:', event.id, { event })
-      return
-    }
-
-    log.debug('getFromOutbox sender found:', event.id, { event, senderPubkey, recipientPubkey })
-
-    const { data } = event
-    const decryptedData = await this.gun
-      .get(senderPubkey)
-      .get('outbox')
-      .get(recipientPubkey)
-      .get(event.id)
-      .onDecrypt()
+    const txData = await this.storage.getFromOutbox(event.id)
+    log.debug('getFromOutbox saved data', txData)
 
-    const updatedEvent = {
+    const updatedEventData = {
       ...event,
-      data: {
-        ...data,
-        ...(decryptedData || {}),
-      },
+      data: { ...event.data, ...txData },
```

Comment:
originally it was txData || {}
did you chcek what happens if there's no txData?

Reply:
Changed in f3caaed8bdbc62f3ba1a4c852c87f10d2168c0df